### DISCUSSION TITLE
Anisotropy

### DISCUSSION THREAD

**Discussion by DP52** [peridigm/peridigm#347] [General]
How to build anisotropic model with peridigm ? 
Can material properties be assigned to Cubit after the anisotropic model is established ? Do I need to redefine the material in the .yaml file after adding material properties to the Cubit ?

**Comment by JTHesse** [comment]
Take a look at [PeriLab.jl](https://github.com/PeriHub/PeriLab.jl). Anisotropic models are implemented and verified there.
